meinberlin/apps/cms/models/storefronts.py

```diff
@@ -84,10 +84,12 @@ def get_district_project_count(self):
                 active_project_count += 1
         return active_project_count
 
-    def save(self, *args, **kwargs):
+    def save(self, update_fields=None, *args, **kwargs):
         if not self.district_project_count:
             self.district_project_count = self.get_district_project_count()
-        super().save(*args, **kwargs)
+            if update_fields:
+                update_fields = {"district_project_count"}.union(update_fields)
+        super().save(update_fields=update_fields, *args, **kwargs)
```

Comment:
@goapunk thanks for the custom save() reminder. I am thinking, this one here, doesn't need always to update the `district_project_count` when the instance is saved? and not only if the count is not set?

Reply:
I'm not completely sure, I didn't write this code but I think the idea is that if it is not set (e.g. when creating the storefront the first time), that it is added here and then it is updated periodically via cron and this management command https://github.com/liqd/a4-meinberlin/blob/main/meinberlin/apps/cms/management/commands/update_storefront_counts.py